interfaces/index.ts

```diff
@@ -1,6 +1,8 @@
 import IEvent from './IEvent';
-import IFlight from './IFlight';
+import { IFlight, Itinerary } from './IFlight';
```

Comment:
Wait what? How are you exporting a part of an interface? Where is Itinerary coming from?

Also if it's an interface, could you please prefix it with 'I'? That way if you import it in a component we later create also called 'Itinerary', it doesn't cause problems.

Reply:
Itinerary was a type, again took from daniele's website. I removed it from the Iflight page and kept your changes, and forgot to remove it from this file (as it didn't show as a conflict)

Reply:
removed